CHANGES.rst

```diff
@@ -15,6 +15,7 @@ Other changes
 - Check for the resultlog by feature and not by version as pytest master does
   not provide a consistent version.
 
+- Add ``condition`` keyword argument to the re-run marker.
```

Comment:
It would be nice, if you could include a link to you github profile.

Reply:
Done. :)